<ui version="4.0" >
 <class>morpherDialog</class>
 <widget class="QWidget" name="morpherDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QPushButton" name="pickSourceMeshButton" >
       <property name="text" >
        <string>Pick Source Mesh</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLabel" name="souceLabel" >
       <property name="text" >
        <string>Souce Mesh Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QPushButton" name="pickDestMeshButton" >
       <property name="text" >
        <string>Pick Dest. Mesh</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLabel" name="destLabel" >
       <property name="text" >
        <string>Dest. Mesh Name</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTreeWidget" name="layerTreeWidget" >
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>10</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
    <property name="columnCount" >
     <number>2</number>
    </property>
    <column>
     <property name="text" >
      <string>1</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>2</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>130</y>
      <width>190</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Start:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="startSpinBox" >
       <property name="minimum" >
        <number>-400</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>End:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="endSpinBox" >
       <property name="maximum" >
        <number>400</number>
       </property>
       <property name="value" >
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="" >
    <property name="geometry" >
     <rect>
      <x>12</x>
      <y>82</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Morph:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sliderValueLabel" >
       <property name="text" >
        <string>000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="morphSlider" >
       <property name="maximum" >
        <number>100</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
